dask/array/tests/test_array_core.py

```diff
@@ -617,6 +617,22 @@ def test_store():
     assert raises(ValueError, lambda: store([at, bt], [at, bt]))
 
 
+def test_to_hdf5():
+    try:
+        import h5py
+    except ImportError:
+        return
```

Comment:
Oh, I'm sorry, I misunderstood.  Yes, I'll give that a shot and see how it works